src/main/java/WebRTCConf.java

```diff
@@ -0,0 +1,244 @@
+import java.util.Arrays;
+import java.util.Random;
+import java.util.Scanner;
+import java.util.logging.Level;
+
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.edge.EdgeOptions;
+import org.openqa.selenium.firefox.FirefoxOptions;
+import org.openqa.selenium.firefox.FirefoxProfile;
+import org.openqa.selenium.safari.SafariOptions;
+
+
+
+public class WebRTCConf {
+   
+   public static int getRoomID() {
+      Random random = new Random();
+        int roomId = random.nextInt(500000);
+        roomId += 100000;
+        
+        return roomId;
+   }
+   
+   public static ChromeOptions getFireFoxConfiguration() {
+      
+      FirefoxProfile firefoxProfile = new FirefoxProfile();
```

Comment:
use formatter for indentation. e.g. Shift + Option + F in VS code.

Reply:
done